components/landing/highlights.json

```diff
@@ -107,7 +107,7 @@
       "title": "Blocking of foreign-based media",
       "text": "Media accused of hostile reporting against the government",
       "report": "https://ooni.io/post/south-sudan-censorship/#blocked-websites",
-      "explore": "/search?input=sudantribune.com&probe_cc=SS",
+      "explore": "/search?domain=sudantritermbune.com&probe_cc=SS",
```

Comment:
This query is missing date ranges. cc @agrabeli.

Reply:
This query also returns zero results.

Reply:
I think there is a typo in the domain name.

Reply:
Ah. My bad.

Reply:
FYI, with the typo fixed, this shows results with or without `www.` prefix to the domain.

Reply:
The correct domain is www.sudantribune.com, which I've added: https://github.com/ooni/explorer/pull/294/files#diff-8c4e91f3eec33fa5a15606b74ddde89a